tests/test_parse.py

```diff
@@ -148,6 +151,7 @@ def test_bad_proto():
 
 @pytest.mark.xfail(reason='working on V2')
 def test_file_json(tmpdir):
+    # TODO: Should the parse_file tests be dropped?
```

Comment:
Question 9: It seems we no longer want to provide any built-in support for parsing files directly; should I just drop all these tests?

Reply:
I would, but maybe the PR that drops them can update the migration guide?

Reply:
yes, remove.